packages/manager/modules/pci/src/projects/project/serving/namespace/tokens/add/translations/Messages_en_GB.json

```diff
@@ -1,6 +1,6 @@
 {
   "pci_projects_project_serving_namespace_tokens_add_error": "An error occur when creating the token : {{ message }}",
-  "pci_projects_project_serving_namespace_tokens_add_success": "Your token is created, save your token <code><pre>{{ token }}</pre></code>",
+  "pci_projects_project_serving_namespace_tokens_add_success": "Your token is created, save your token",
```

Comment:
Once again, those translations will be deleted :)

Reply:
Yeah I know, professional conscience I guess